scripts/travis/build-all-in-one-image.sh

```diff
@@ -26,42 +29,89 @@ run_integration_test() {
   docker kill $CID
 }
 
-upload_to_docker() {
-  # Only push the docker container to Docker Hub for master branch
-  if [[ ("$BRANCH" == "master" || $BRANCH =~ ^v[0-9]+\.[0-9]+\.[0-9]+$) && "$TRAVIS_SECURE_ENV_VARS" == "true" ]]; then
-    echo 'upload to Docker Hub'
+docker_buildx() {
+  CMD_ROOT=${1}
+  FLAGS=${@:2}
+
+  if [[ "$FLAGS" == *"-push"*  ]]; then
+    if [[ ("$BRANCH" == "master" || $BRANCH =~ ^v[0-9]+\.[0-9]+\.[0-9]+$) && "$TRAVIS_SECURE_ENV_VARS" == "true" ]]; then
+      echo 'upload to Docker Hub'
+    else
+      echo 'skip docker upload for PR'
+      exit 0
+    fi
+  fi
+
+  docker buildx build -f ${CMD_ROOT}/Dockerfile ${FLAGS} ${CMD_ROOT}
+}
+
+image_tags_for() {
+  REPO=${1}
+
+  major=""
+  minor=""
+  patch=""
+
+  if [[ "$BRANCH" == "master" ]]; then
+    TAG="latest"
+  elif [[ $BRANCH =~ ^v([0-9]+)\.([0-9]+)\.([0-9]+)$ ]]; then
+    major="${BASH_REMATCH[1]}"
+    minor="${BASH_REMATCH[2]}"
+    patch="${BASH_REMATCH[3]}"
+    TAG=${major}.${minor}.${patch}
   else
-    echo 'skip docker upload for PR'
-    exit 0
+    TAG="${BRANCH///}"
+  fi
+
+  IMAGE_TAGS="--tag ${REPO}:${TAG}"
```

Comment:
@aebirim, for your change, you might just need to add a new tag here, as well as a `docker login` to all configured repositories.

Reply:
> @aebirim, for your change, you might just need to add a new tag here, as well as a `docker login` to all configured repositories.

np. @albertteoh @jpkrohling  I'll wait for this script to be merged before adding my quay change

Reply:
This PR was closed in favor of #2611